src/components/icons/index.ts

```diff
@@ -102,3 +102,7 @@ export { PickUpIcon } from './PickUpIcon';
 export { InformationIcon } from './InformationIcon';
 export { PartialClockIcon } from './PartialClockIcon';
 export { LockIcon } from './LockIcon';
+export { PackageBasicIcon } from './PackageBasicIcon';
+export { PackagePlusIcon } from './PackagePlusIcon';
+export { PackagePremiumIcon } from './PackagePremiumIcon';
+export { PackageUnlimitedIcon } from './PackageUnlimitedIcon';
```

Comment:
There might be an opportunity to improve icon names. Instead of naming them by their usage (package, basic, etc), we could name them by what they actually are (eg. `TiltedRectangle`, `PlusCircle`, `StarCircle`, `InfinityCircle` or something similar). That way if we re-use them outside the context of packages, the names might make more sense. WDYT?

Reply:
This naming is much better in terms of reusability. Working on it.